openapi/openapi.yaml

```diff
@@ -40,7 +40,7 @@ tags:
 paths:
   /agencies:
     $ref: paths/agencies.yaml
-  /agencyBugdets:
+  /agency-bugdets:
```

Comment:
Still says bu**gd**et, should be bu**dg**et.

Reply:
Fixed, sorry about that.